force-app/main/default/objects/npe01__OppPayment__c/fields/DebitType__c.field-meta.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
-    <fullName>RefundType__c</fullName>
-    <description>Determine if the record is fully or partially refunded</description>
+    <fullName>DebitType__c</fullName>
+    <description>Indicates the type of debit.</description>
```

Comment:
Will this be clear to users? Specifically that by "debit" we mean a form of a refund? I haven't seen that term used anywhere else. I couldn't find it in any labels either.

Reply:
We updated it to Debit so it can cover the future Chargeback use case too. It's not the most friendly term, but it is common accounting term.